test/init_test_env.sh

```diff
@@ -56,8 +56,4 @@ echo "Starting local helm server"
 curl -s 'http://localhost:52000/helm'
 helm repo update > /dev/null
 echo "Initialization done"
-
-echo "Adding SLATE helm repository"
-helm repo add slate-dev https://jenkins.slateci.io/catalog/incubator/ 
-helm repo add slate-prod https://jenkins.slateci.io/catalog/stable/
-echo "Repository added successfully"
+helm repo add local http://localhost:8879
```

Comment:
It might be nice to move this line before 58, so that everything is done before the print statement claims so, but it's hard to imagine a situation where this would truly matter.

Reply:
I moved the line before the print statement. Thanks for the suggestion, Chris